doc/openocd.texi

```diff
@@ -11524,6 +11524,20 @@ as in the mie CSR (defined in the RISC-V Privileged Spec).
 For details on this trigger type, see the RISC-V Debug Specification.
 @end deffn
 
+@deffn {Command} {riscv reserve_trigger} [index [@option{on|off}]]
```

Comment:
Looking at the code of `COMMAND_HANDLER(handle_reserve_trigger)`, the second argument (`on|off`) is actually mandatory:

```suggestion
@deffn {Command} {riscv reserve_trigger} [index @option{on|off}]
```

Reply:
Addressed.